resources/views/custom_fields/fieldsets/view.blade.php

```diff
@@ -75,7 +75,6 @@
                 'id' => 'ordering']) }}
                 {{ Form::checkbox("required","on") }}
                 {{ trans('admin/custom_fields/general.required') }}
-                {{ Form::text("order",$maxid)}}
```

Comment:
This looks like your older commits from the other PR are in here again. Each PR should resolve a single issue when possible. (It does sometimes happen that you get a twofer because the issues are closely related, but generally you want to make a new feature branch off of `develop` or `master` each time you're addressing a new PR.